web/src/components/VirtualTable.tsx

```
@@ -0,0 +1,172 @@
+                        <TooltipContent>
+                          {video.video_title}
+                        </TooltipContent>
+                        <TooltipTrigger>
+                        <h3 className="text-start font-medium text-sm leading-tight truncate sm:w-[150px] lg:w-[200px] xl:w-[250px] 2xl:w-[300px]">
+                          {video.video_title}
+                        </h3>
+                        </TooltipTrigger>
+                      </Tooltip>
+                      <div className="flex flex-wrap gap-1">
+                        {video.detailed_subtags.split(',').slice(0, 2).map((tag, i) => (
+                            <Badge variant="secondary" className="text-xs max-w-20 truncate">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)